rust/src/detect/error.rs

```diff
@@ -23,6 +23,7 @@ use nom7::error::{ErrorKind, ParseError};
 #[derive(Debug, PartialEq, Eq)]
 pub enum RuleParseError<I> {
     InvalidByteMath(String),
+    InvalidTransformBase64(String),
```

Comment:
This feels weird to me, cc @jasonish what do you think of it ?

Reply:
Seems to be at the same scope and usage pattern as InvalidByteMath, so seems fine.  Would be nice to document whats in that string, for bytemath as well though.